tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithListView.spec.ts

```diff
@@ -0,0 +1,348 @@
+  // Checks if child is published
+  const childContentData = await umbracoApi.document.getByName(childContentName);
+  expect(childContentData.variants[0].state).toBe(expectedState);
+});
+
+test('can not publish child in a list when parent is not published', async ({umbracoApi, umbracoUi}) => {
+  // Arrange
+  const expectedState = 'Draft';
+  const childDocumentTypeId = await umbracoApi.documentType.createDefaultDocumentType(childDocumentTypeName);
+  await umbracoApi.dataType.createListViewContentDataType(dataTypeName);
+  const dataTypeData = await umbracoApi.dataType.getByName(dataTypeName);
+  const documentTypeId = await umbracoApi.documentType.createDocumentTypeWithAPropertyEditorAndAnAllowedChildNode(documentTypeName, dataTypeName, dataTypeData.id, childDocumentTypeId);
+  const documentId = await umbracoApi.document.createDefaultDocument(contentName, documentTypeId);
+  await umbracoApi.document.createDefaultDocumentWithParent(childContentName, childDocumentTypeId, documentId);
+  await umbracoUi.content.goToSection(ConstantHelper.sections.content);
+  await umbracoUi.content.goToContentWithName(contentName);
+
+  // Act
+  await umbracoUi.content.goToContentInListViewWithName(childContentName);
+  await umbracoUi.content.clickContainerSaveAndPublishButton();
+
+  // Assert
+  // Content created, but not published
+  await umbracoUi.content.doesSuccessNotificationsHaveCount(1);
+  await umbracoUi.content.isErrorNotificationVisible();
+  const contentData = await umbracoApi.document.getByName(contentName);
+  expect(contentData.variants[0].state).toBe(expectedState);
+  expect(await umbracoApi.document.getChildrenAmount(documentId)).toEqual(1);
+  // Checks if child is still in draft
+  const childContentData = await umbracoApi.document.getByName(childContentName);
+  expect(childContentData.variants[0].state).toBe(expectedState);
+});
+
+test('child is removed from list after child content is deleted', async ({umbracoApi, umbracoUi}) => {
+  // Arrange
+  const childDocumentTypeId = await umbracoApi.documentType.createDefaultDocumentType(childDocumentTypeName);
+  await umbracoApi.dataType.createListViewContentDataType(dataTypeName);
+  const dataTypeData = await umbracoApi.dataType.getByName(dataTypeName);
+  const documentTypeId = await umbracoApi.documentType.createDocumentTypeWithAPropertyEditorAndAnAllowedChildNode(documentTypeName, dataTypeName, dataTypeData.id, childDocumentTypeId);
+  const documentId = await umbracoApi.document.createDefaultDocument(contentName, documentTypeId);
+  await umbracoApi.document.createDefaultDocumentWithParent(childContentName, childDocumentTypeId, documentId);
+  await umbracoUi.content.goToSection(ConstantHelper.sections.content);
+  expect(await umbracoApi.document.getChildrenAmount(documentId)).toEqual(1);
+  await umbracoUi.content.goToContentWithName(contentName);
+  await umbracoApi.document.ensureNameNotExists(childContentName);
```

Comment:
This should be in the Act section and should interact via the GUI instead of the API, as it is not precondition step.